Makefile

```diff
@@ -109,6 +116,11 @@ $(RECEPTORCTL_WHEEL): receptorctl/README.md receptorctl/setup.py $(shell find re
 
 receptorctl_wheel: $(RECEPTORCTL_WHEEL)
 
+# Build and install receptorctl into your python environment
+receptorctl_install: $(RECEPTORCTL_WHEEL)
```

Comment:
I question the general usefulness of these types of targets. Makefiles are for building software and executing commands. I would consider it an anti-pattern to have targets that alter the user's environment. I would like to see this removed.